reader-gtfs/src/main/java/com/graphhopper/reader/gtfs/RealtimeFeed.java

```diff
@@ -257,6 +257,11 @@ public Graph copyTo(Graph g) {
             public GraphExtension getExtension() {
                 throw new RuntimeException();
             }
+
+            @Override
+            public int getBaseNode(int edge, int adjNode) {
+                return 0;
```

Comment:
or better throw `RuntimeException` like in `getExtension` ? Btw have you considered adding mockito to core ?

Reply:
> or better throw `RuntimeException` like in `getExtension` ?

Yes

> Btw have you considered adding mockito to core ?
You mean in the sources directly? I would strongly argue against it :) (It does too much magic, hard to debug, not good for portability etc)

Reply:
oh right this is production code. but for tests I like mockito (just not power mockito) ;)

Reply:
i replaced with exception